rpcd/playbooks/ansible.cfg

```diff
@@ -1,25 +1,4 @@
 [defaults]
-gathering = smart
-fact_caching = jsonfile
-fact_caching_connection = /etc/openstack_deploy/ansible_facts
-fact_caching_timeout = 86400
-host_key_checking = False
-
-# Setting forks should be based on your system. The Ansible defaults to 5,
-# the os-lxc-hosts assumes that you have a system that can support
-# OpenStack, thus it has been conservatively been set to 15
-forks = 15
-
-# SSH timeout
+# NOTE: The ssh timeout is increased here to 120 (OSA's default is 5) to
+#       avoid transient ssh errors in some environments.
 timeout = 120
```

Comment:
Not sure exactly what they were, but I remember @andymcc proposed those patches. We had some ssh connectivity issues to newly built containers and increasing the timeouts got us over the hump.